### ISSUE TITLE
I get a LOT of bookmarks shown when selecting multiple verses

### ISSUE THREAD

**Issue by agrogers** [AndBible/and-bible#1476]
I noticed this yesterday. I selected multiple verses to add a bookmark.  The dialog showed me a lot more bookmarks than i had expected to see. I think it works as expected - all bookmarks in all selected verses are shown. But it was a bit overwhelming and possibly not so useful.

This is a different issue to the one we discussed where we show all bookmarks for the verse not just the bookmark the user clicked on. In that issue we could hide the extra bookmarks in a 'More...' group. 

This scenario requires that all of them are shown. Maybe this is a good feature. I am not sure yet :) Just interested to hear what others think.

![image](https://user-images.githubusercontent.com/13920678/127799004-c9ae7b2f-8885-43bc-a441-3a48bb33a59b.png)

**Comment by bvahedy** [comment]
I kind of like it as long as the color is the bookmarks match what is on
the verse. This way The user can scroll through and take action on a
specific item. If we use "more" to hide lables,  it not only creates one
more click but also the user may never realize that  lables are hidden
under "more".

**Comment by tuomas2** [comment]
This will be fixed by the suggestions you have raised and I believe this is not release-critical usability issue?

**Comment by agrogers** [comment]
No, definitely not important. I'm not sure that anything I have proposed
managed this situation. And I am not sure this should be changed. Just
flagging it.
